client/src/main/java/com/paypal/selion/reports/reporter/html/HtmlReporterListener.java

```diff
@@ -300,11 +300,13 @@ private String getContent(ITestResult result) {
                     // Attaching ralogId to each of the page title.
                     if ((logLine.getHref() != null) && (logLine.getHref().length() > 1)) {
                         htmllog = "<a href='../" + logLine.getHref() + "' title='" + logLine.getLocation() + "' >"
-                                + logLine.getMsg() + "</a>";
+                                 + "Page Source </a>";
```

Comment:
This change seems introduces a breakage in the older Html Report. 

If I call;

```java 
SeLionReporter.log("foo",true,true)
```
The link text should be `foo`.. With the change, the link text will Always be `Page Source`.